Dockerfile

```diff
@@ -9,6 +9,8 @@ COPY package*.json /usr/src/kort/
 # the --only=production flag means this will not install the devDependencies
 RUN npm install --only=production
 COPY . /usr/src/kort
+# build browserify bundle 
+RUN npm run build 
```

Comment:
Shouldn't  `npm run build` be called before copy, otherwise the bundle file will not get copied over

Reply:
I also thought that but copying the files before running the npm run build command means the files needed(uuid-helper.js) are available for the build

Reply:
However, this has raised a point that the script needs another copy for the created files.

Reply:
ah right, ignore me